app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_with model: post, url: user_posts_path do |form| %>
+  <% if post.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% post.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+    <%= form.hidden_field :author_id%>
+    <%= form.hidden_field :comments_counter%>
+    <%= form.hidden_field :likes_counter%>
```

Comment:
- [ ] As requested by the previous reviewer, The comments_counter, likes_counter, and author_id should not be input. Kindly remove these fields, and add the comments_counter, likes_counter, and author_id in the controller instead. As I said in the previous comment, you may assign current_user for author_id.

Reply:
@medaminedev66, how can I implement this?